### DISCUSSION TITLE
Get LiveReload to work with custom hostnames

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [remix-run/remix#6998] [Q&A] [answered]
I have a custom hostname set up for my project with reverse proxy. 
I'm now able to run the app from https://myhost.local successfully. 

The only thing I struggle with is live reloading which is not working. I'm aware there's a second server running in Remix that handles reloading, but I have no idea how to set it up to work with custom hostname.

There's a similar [stale issue](https://github.com/remix-run/remix/issues/4476) already reported, but I'm wondering if anyone have any solution (could be for v1 or v2 dev server).

Thanks!

**Comment by kiliman** [comment, marked as answer]
There's a new env variable, `REMIX_DEV_ORIGIN`, that the `LiveReload` component uses.

So set `REMIX_DEV_ORIGIN=https://myhost.local`

https://github.com/remix-run/remix/blob/47a98bc78d6b5a48de11b6138003c30842b59b37/packages/remix-react/components.tsx#L1830

**Comment by chinanderm** [comment]
@piotrkulpinski Would you be able to share how you've set up the reverse proxy? I'm using [local-ssl-proxy](https://www.npmjs.com/package/local-ssl-proxy) but the `request` that comes into Remix is `http` instead of `https`.

**Reply by piotrkulpinski** [reply]
I'm using the [LocalCan](https://www.localcan.com) app for MacOS.
